<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuCrearPartida</class>
 <widget class="QMainWindow" name="MenuCrearPartida">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Worms 2D</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow { 
	background-image: url(&quot;../client/vista/data/fondo.png&quot;); 
	background-position: center center;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="nuevaPartidaButton">
       <property name="geometry">
        <rect>
         <x>158</x>
         <y>400</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: black;
	border-style: inset;
	border-width: 3px;
	border-color: grey;
	font: bold 14px;
	color: grey;
}

QPushButton:pressed {
	background-color: grey;
	border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Crear partida</string>
       </property>
      </widget>
      <widget class="QComboBox" name="jugadoresComboBox">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>200</y>
         <width>180</width>
         <height>40</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
	background-color: black;
	border-style: inset;
	border-width: 3px;
	border-color: grey;
	font: bold 14px;
	color: grey;
}

QComboBox QAbstractItemView {
	background-color: black;
	border-style: outset;
	border-width: 3px;
	border-color: grey;
	font: bold 14px;
	color: black;
	border: 2px solid grey;
}</string>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string>Seleccione una opcion</string>
       </property>
       <item>
        <property name="text">
         <string>Seleccione una opcion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="mapaComboBox">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>250</y>
         <width>180</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
	background-color: black;
	border-style: inset;
	border-width: 3px;
	border-color: grey;
	font: bold 14px;
	color: grey;
}

QComboBox QAbstractItemView {
	background-color: black;
	border-style: outset;
	border-width: 3px;
	border-color: grey;
	font: bold 14px;
	color: black;
	border: 2px solid grey;
}</string>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string>Seleccione una opcion</string>
       </property>
       <item>
        <property name="text">
         <string>Seleccione una opcion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jaula grande</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jaula peque√±a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Puente</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vigas inclinadas</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="volverAtrasButton">
       <property name="geometry">
        <rect>
         <x>332</x>
         <y>400</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: black;
	border-style: inset;
	border-width: 3px;
	border-color: grey;
	font: bold 14px;
	color: grey;
}

QPushButton:pressed {
	background-color: grey;
	border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Volver atras</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelJugadores">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>210</y>
         <width>171</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: grey;
font: 16px;</string>
       </property>
       <property name="text">
        <string>Cantidad de jugadores</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelMapa">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>260</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: grey;
font: 16px;</string>
       </property>
       <property name="text">
        <string>Mapa</string>
       </property>
      </widget>
      <zorder>nuevaPartidaButton</zorder>
      <zorder>jugadoresComboBox</zorder>
      <zorder>mapaComboBox</zorder>
      <zorder>labelJugadores</zorder>
      <zorder>labelMapa</zorder>
      <zorder>volverAtrasButton</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
